layouts/partials/css.html

```diff
@@ -21,5 +21,7 @@
 <!-- Google Fonts - Ubuntu -->
 <link href="https://fonts.googleapis.com/css?family=Ubuntu&display=swap" rel="stylesheet">
 <link href="https://fonts.googleapis.com/css?family=Rubik&display=swap" rel="stylesheet">
```

Comment:
If you are using Lato font now then can we remove Rubik font link ?

Reply:
Yes, we can.

I've been playing around with fonts for the main hero title. I'm not sure Ubuntu is the best choice here, although I do think it looks quite nice for body text.

I'll write more soon in #48.